.gitmodules

```
@@ -0,0 +1,3 @@
+[submodule "dags/GOB-Prepare"]
+	path = dags/GOB-Prepare
+	url = https://github.com/Amsterdam/GOB-Prepare.git
```

```suggestion
	url = https://github.com/Amsterdam/GOB-Prepare.git
	branch = master
	shallow = true
	fetchRecurseSubmodules = true
```

https://www.git-scm.com/docs/gitmodules

iets explicieter zo denk ik